docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+     - Not relevant
+     -
+   * - `3.3.3.6. Creating the class object <https://docs.python.org/3/reference/datamodel.html#creating-the-class-object>`_
+     - Not relevant
+     -
+   * - `3.3.3.7. Uses for metaclasses <https://docs.python.org/3/reference/datamodel.html#uses-for-metaclasses>`_
+     - Not relevant
+     -
+   * - `3.3.4. Customizing instance and subclass checks <https://docs.python.org/3/reference/datamodel.html#customizing-instance-and-subclass-checks>`_
+     - Not Supported
+     -
+   * - `3.3.5. Emulating generic types <https://docs.python.org/3/reference/datamodel.html#emulating-generic-types>`_
+     - Not Supported
+     -
+   * - `3.3.6. Emulating callable objects <https://docs.python.org/3/reference/datamodel.html#emulating-callable-objects>`_
+     - Supported
+     -
+   * - `3.3.7. Emulating container types <https://docs.python.org/3/reference/datamodel.html#emulating-container-types>`_
+     - Partially Supported
+     - some magic methods not supported (e.g. **iter**\ )
+   * - `3.3.8. Emulating numeric types <https://docs.python.org/3/reference/datamodel.html#emulating-numeric-types>`_
+     - Partially Supported
+     - magic methods with swapped operands not supported (\ **r***\ )
+   * - `3.3.9. With Statement Context Managers <https://docs.python.org/3/reference/datamodel.html#with-statement-context-managers>`_
+     - Not Supported
+     -
+   * - `3.3.10. Special method lookup <https://docs.python.org/3/reference/datamodel.html#special-method-lookup>`_
+     - Not relevant
+     -
+   * - `3.4. Coroutines <https://docs.python.org/3/reference/datamodel.html#coroutines>`_
+     - Not Supported
+     -
+   * - `3.4.1. Awaitable Objects <https://docs.python.org/3/reference/datamodel.html#awaitable-objects>`_
+     - Not Supported
+     -
+   * - `3.4.2. Coroutine Objects <https://docs.python.org/3/reference/datamodel.html#coroutine-objects>`_
+     - Not Supported
+     -
+   * - `3.4.3. Asynchronous Iterators <https://docs.python.org/3/reference/datamodel.html#asynchronous-iterators>`_
+     - Not Supported
+     -
+   * - `3.4.4. Asynchronous Context Managers <https://docs.python.org/3/reference/datamodel.html#asynchronous-context-managers>`_
+     - Not Supported
+     -
+   * - `4. Execution model <https://docs.python.org/3/reference/executionmodel.html#>`_
+     - Not Relevant
+     - Not directly relevant but this could use some explanation on how TorchScript differs from Python
+   * - `4.1. Structure of a program <https://docs.python.org/3/reference/executionmodel.html#structure-of-a-program>`_
+     - Not Relevant
+     -
+   * - `4.2. Naming and binding <https://docs.python.org/3/reference/executionmodel.html#naming-and-binding>`_
+     - Not Relevant
+     - Names are entriely bound at compile time in TorchScript
+   * - `4.2.1. Binding of names <https://docs.python.org/3/reference/executionmodel.html#binding-of-names>`_
+     - Not Relevant
+     - See ``global`` and ``nonlocal`` statements
```

Comment:
What does this means `See ``global`` and ``nonlocal`` statements` ?  if it's a link to another section can u add the ref